src/translators/LLMTranslators/GeminiLLMTranslator/index.ts

```diff
@@ -0,0 +1,34 @@
+import { GeminiLLMFetcher, GeminiLLMFetcherOptions } from './GeminiLLMFetcher';
+import { LLMTranslator } from '../LLMTranslator';
+import { BaseLLMTranslatorConfig } from '..';
+
+type GeminiLLMTranslatorConfig = BaseLLMTranslatorConfig<GeminiLLMFetcherOptions> & {
+	apiKey: string;
+};
+
+export class GeminiLLMTranslator extends LLMTranslator {
+	public static readonly translatorName: string = 'GeminiLLMTranslator';
+
+	constructor({
+		apiKey,
+		llmFetcherOptions,
+		translatorOptions,
+	}: GeminiLLMTranslatorConfig) {
```

Comment:
Improve constructor config API.

As user i want to pass some options for every instance i create. Options like API key, prompt generator, model name and similar will be used frequently.

Current API is confuse me because i have to pass this options under specific sections of config.

Instead, this options must be in root and you should map it manually in constructor body